devtools/aesh/reflectconfigs/quarkuscli.json

```diff
@@ -6,7 +6,8 @@
     "allDeclaredMethods" : true,
```

Comment:
someone should make a quarkus aesh extension making this not needed :)

Reply:
I agree :)